content/word-lists/tier-1.md

```diff
@@ -6,25 +6,45 @@ Terms in this list should be replaced immediately when encountered. Read the [Ov
 
 ## Whitelist, blacklist
 
-**Definition:** 
+### Terms
 
-Lists which permit or deny a set of nouns, or select enabled features.
+* blacklist
+* whitelist
 
-**Proposed alternative:** 
+### Concept
 
-* allowlist/denylist 
-* allowedNouns/deniedNouns
+Noun: A list of authorized or unauthorized entities that might be permitted or denied access to computers and networks.
 
-**Reasoning:** 
+Verb: The computer administration practice of creating a list of authorized or unauthorized entities to protect computers and networks and to prevent unnecessary demand for resources.
 
-The underlying assumption of the whitelist/blacklist metaphor is that white = good and black = bad. 
-Because colors in and of themselves have no predetermined meaning, any meaning we assign to them is cultural: for example, the color red in many Southeast Asian countries is lucky, and is often associated with events like marriages, whereas the color white carries the same connotations in many European countries. 
-In the case of whitelist/blacklist, the terms originate in the publishing industry – one dominated by the USA and England, two countries which participated in slavery and which grapple with their racist legacies to this day.
+### Recommendation
 
-From a technical communication perspective, using whitelist/blacklist as a naming convention applies metaphor (and, in turn, unintended meaning) when it isn’t needed. 
-More directly descriptive words like allowlist/denylist enhances understanding. Allowlist/denylist, or simply using allowed/denied as an entity prefix has the added benefit of being easily translatable to other human languages.
+Replace immediately.
 
-**Recommendation:** Adopt immediately
+### Recommended replacement terms
+
+**Preferred replacement terms**
+
+* blocklist/denylist/block
+* allowlist/allow
+
+**Admitted replacement terms**
+
+* passlist
+* deny
+
+### Unsuitable replacement terms
+
+### Use context
+
+* blocklist or denylist: A list of entities that are denied access
+* allowlist or passlist: A list of entities that are permitted access
+
+### Rationale for replacement
+
+These terms are based on a metaphor where black represents bad and white represents good, which reinforces systemic racial bias. Terms that use color representationally, such as “black box” and “white glove,” do not reinforce this discriminatory metaphor and are fine to use.
+
+From a technical communication perspective, using whitelist/blacklist as a naming convention applies metaphor (and, in turn, unintended meaning) when it isn’t needed. More directly descriptive words like blocklist/allowlist enhances understanding.
```

Comment:
Change "enhances" > "enhance", for subject-verb agreement.
Alternatively: "The use of more directly descriptive words ... enhances understanding."

Reply:
```suggestion
From a technical communication perspective, using whitelist/blacklist as a naming convention applies metaphor (and, in turn, unintended meaning) when it isn’t needed. More directly descriptive words like blocklist/allowlist enhance understanding.
```